etc/db-config.yaml

```
@@ -215,6 +215,11 @@
             default_value: false
             public: true
             description: Should teams be able to view a diff of their and the reference output on sample testcases?
+        -   name: show_too-late_result
```

I would call it 
```suggestion
        -   name: show_too_late_result
```

to be consistent with other option naming.